downstream/modules/platform/con_storage-options-for-operator-installation-on-ocp.adoc

```diff
@@ -0,0 +1,14 @@
+[id="con_storage-options-for-operator-installation-on-ocp_{context}"]
+= Storage options for {OperatorPlatform} installation on {OCP}
+
+If you are using file-based storage and your installation scenario includes {HubName}, you must ensure that your storage option for {OperatorPlatform} has been set to `ReadWriteMany`.
+
+{HubNameStart} requires `ReadWriteMany` storage for operation or Azure Blob storage or an S3-compliant storage because multiple pods responsible for upload and download of content (for example, collections) need access to the same content. Otherwise, multiple pods are not guaranteed to have access to the same persistent volume claim (PVC)  unless it is `ReadWriteMany`.
+
+In addition, OpenShift Data Foundation provides a `ReadWriteMany` or s3-compliant implementation. Also, an NFS storage configuration can be setup to support `ReadWriteMany` and is often the simplest approach. This, however, introduces the NFS server as a potential single point of failure.
```

Comment:
Is S3 supposed to be uppercase or lowercase?

Reply:
Do you think it would be useful to have an attribute for OpenShift Data Foundation?

Reply:
I actually considered that. I have added two entries to the attributes file for this: 

:ODF: Red Hat OpenShift Data Foundation
:ODFShort: OpenShift Data Foundation